main.component.scss

```diff
@@ -0,0 +1,51 @@
+@import "../variables.scss";
+
+div{

```

Comment:
these styles will affect all divs in component, replace selector with class with respective name